qiskit/providers/aer/pulse_new/models/operator_models.py

```diff
@@ -0,0 +1,409 @@
+            signal_mapping (Callable): a function returning wither a
+                                       VectorSignal or a list of Signal objects.
+
+            frame_operator (Operator): Rotating frame operator.
+            cutoff_freq (float): Frequency cutoff when evaluating the model.
+        """
+
+        self._operators = operators
+
+        self._frame_operator = frame_operator
+        self._cutoff_freq = cutoff_freq
+
+        # initialize signals
+        self._signal_params = None
+        self._signals = None
+        self._carrier_freqs = None
+        self._signal_mapping = signal_mapping
+
+        if signals is not None:
+            # note: setting signals includes a call to _construct_frame_helper()
+            self.signals = signals
+        else:
+            self._construct_frame_helper()
+
+    @property
+    def signals(self) -> VectorSignal:
+        """Return the signals in the model"""
+        return self._signals
+
+    @signals.setter
+    def signals(self, signals):
+        """Set the signals.
+
+        Behavior:
+            - If no signal_mapping is specified, the argument signals is
+              assumed to be either a list of Signal objects, or a VectorSignal,
+              and is saved in self._signals.
+            - If a signal_mapping is specified, signals is assumed to be a valid
+              input of signal_mapping. The argument signals is set to
+              self._signal_params, and the output of signal_mapping is saved in
+              self._signals.
+        """
+        if signals is None:
+            self._signal_params = None
+            self._signals = None
+            self._carrier_freqs = None
+        else:
+
+            # if a signal_mapping is specified, take signals as the input
+            if self._signal_mapping is not None:
+                self._signal_params = signals
+                signals = self._signal_mapping(signals)
+
+            # if signals is a list, instantiate a VectorSignal
+            if isinstance(signals, list):
+                signals = VectorSignal.from_signal_list(signals)
+
+            # if it isn't a VectorSignal by now, raise an error
+            if not isinstance(signals, VectorSignal):
+                raise Exception('signals specified in unaccepted format.')
+
+            # check if the new carrier frequencies are different from the old.
+            # if yes, update them and reinstantiate the frame helper.
+            new_freqs = signals.carrier_freqs
+            if any(signals.carrier_freqs != self._carrier_freqs):
+                self._carrier_freqs = signals.carrier_freqs
+                self._construct_frame_helper()
+
+            self._signals = signals
+
+    @property
+    def frame_operator(self):
+        """Return the frame operator."""
+        return self._frame_operator
+
+    @frame_operator.setter
+    def frame_operator(self, frame_operator):
+        """Set the frame operator; must be anti-Hermitian.
```

Comment:
Can we spend a few more lines on what the frame operator is and what it does?

Reply:
I added a sentence pointing to the class doc string for this, where the effects of setting a frame operator are generally explained.